index.js

```diff
@@ -18,5 +19,17 @@ module.exports = (red, green, blue) => {
 		throw new TypeError('Expected three numbers below 256');
 	}
 
-	return ((blue | green << 8 | red << 16) | 1 << 24).toString(16).slice(1);
+	if (typeof alpha === 'number') {
+		if (alpha < 1 && alpha >= 0) {
+			alpha = Math.round(256 * alpha).toString(16);
```

Comment:
I tried to not use strings, but we run into issues when working with more than 32 bits

```js
let red = 0,
  green = 0,
  blue = 0,
  result = ((blue | green << 8 | red << 16) | 1 << 24).toString(16);
// result = '1000000'; <- the "1" is removed by the slice(1)

```

Now if I didn't use strings for the alpha we're beyond 32 bits and the value becomes zero:

```js
let red = 0,
  green = 0,
  blue = 0,
  rgb = (blue | green << 8 | red << 16) | 1 << 24,
  alpha = 100;
result = (rgb << 8 | Math.round(256 * alpha / 100)).toString(16);
// result = '100';
```

So, I think using strings keeps everything readable, and it works.